demos/interactive/wizard.php

```diff
@@ -27,38 +27,38 @@
 // form on "Next" button click, performing validation and submission. You do not need
 // to return any action from form's onSubmit callback. You may also use memorize()
 // to store wizard-specific variables
-$t->addStep(['Set DSN', 'icon' => 'configure', 'description' => 'Database Connection String'], function (Wizard $w) {
-    $f = \atk4\ui\Form::addTo($w);
+$wizard->addStep(['Set DSN', 'icon' => 'configure', 'description' => 'Database Connection String'], function (Wizard $wizard) {
+    $form = \atk4\ui\Form::addTo($wizard);
     // IMPORTANT - needed for php_unit Wizard test.
-    $f->name = 'w_form';
+    $form->name = 'w_form';
 
-    $f->addControl('dsn', 'Connect DSN', ['required' => true])->placeholder = 'mysql://user:pass@db-host.example.com/mydb';
-    $f->onSubmit(function (\atk4\ui\Form $form) use ($w) {
-        $w->memorize('dsn', $form->model->get('dsn'));
+    $form->addControl('dsn', 'Connect DSN', ['required' => true])->placeholder = 'mysql://user:pass@db-host.example.com/mydb';
+    $form->onSubmit(function (\atk4\ui\Form $form) use ($wizard) {
+        $wizard->memorize('dsn', $form->model->get('dsn'));
 
-        return $w->jsNext();
+        return $wizard->jsNext();
     });
 });
 
 // Alternatvely, you may access buttonNext , buttonPrev properties of a wizard
 // and set a custom js action or even set a different link. You can use recall()
 // to access some values that were recorded on another steps.
-$t->addStep(['Select Model', 'description' => '"Country" or "Stat"', 'icon' => 'table'], function (Wizard $w) {
+$wizard->addStep(['Select Model', 'description' => '"Country" or "Stat"', 'icon' => 'table'], function (Wizard $w) {
     if (isset($_GET['name'])) {
         $w->memorize('model', $_GET['name']);
         $w->app->redirect($w->urlNext());
     }
 
     $c = \atk4\ui\Columns::addTo($w);
```

Comment:
ideally fix all one char variables related with components like this

Reply:
fixed